### ISSUE TITLE
Does not seems to work with Expo Cli

### ISSUE THREAD

**Issue by selimbenmrad** [barmej/react-native-youtube-player#29]
Hi, it seems really helpful, thanks for the person who has shared it, but I was wondering if it does work with expo Cli,  I do not know how to do the linking part with expo Cli?

**Comment by yjose** [comment]
Yeah, unfortunately the package will not work with expo as we need react-native-orientation package

**Comment by butri71** [comment]
any fix for this planned any time soon?

**Comment by yjose** [comment]
Not sure if you can do it  As using `react-native-orientation` is essential for the fullscreen feature.
Any idea if expo has some alternative for `react-native-orientation`?

**Comment by butri71** [comment]
to be honest I'm not using orientation at all in my project, sorry can't help.

**Comment by ger88555** [comment]
Expo now provides an alternative: [`screen-orientation`](https://docs.expo.dev/versions/latest/sdk/expo-screen-orientation/)
